src/App.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <img alt="Vue logo" src="./assets/logo.png" />
+  <h1>JSON Forms Vue 3 Typescript</h1>
+  <json-forms
+    v-bind:data="data"
+    v-bind:renderers="renderers"
+    v-bind:schema="schema"
+    v-bind:uischema="uischema"
+    @change="onChange"
+  />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import { JsonForms, JsonFormsChangeEvent } from "@jsonforms/vue";
+import { defaultStyles, mergeStyles, vanillaRenderers } from '@jsonforms/vue-vanilla'
+
+// mergeStyles combines all classes from both styles definitions into one
+const myStyles = mergeStyles(defaultStyles, { control: { label: 'mylabel', } });
+
+const renderers = [
+  ...vanillaRenderers,
+  // here you can add custom renderers
+];
+
+const uischema = {
+  type: "VerticalLayout",
+  elements: [
+    {
+      type: "Control",
+      scope: "#/properties/name",
+    },
+    {
+      type: "Control",
+      scope: "#/properties/description",
+    },
+    {
+      type: "Control",
+      scope: "#/properties/done",
+    },
+    {
+      type: "Control",
+      scope: "#/properties/dueDate",
+    },
+    {
+      type: "Control",
+      scope: "#/properties/rating",
+    },
+    {
+      type: "Control",
+      scope: "#/properties/recurrence",
+    },
+    {
+      type: "Control",
+      scope: "#/properties/recurrenceInterval",
+    },
+  ],
```

Comment:
Maybe use a horizontal with two verticals so it looks a little bit more fancy ;)